M/Mineos/build_tarballs.jl

```diff
@@ -3,19 +3,32 @@
 using BinaryBuilder
 
 name = "Mineos"
-version = v"1.0"
+version = v"1.0.2" 
 
 # Collection of sources required to build Mineos
 sources = [
-    "https://github.com/anowacki/mineos.git" =>
-    "e2558b486d7656ef112608a8776643da66dc87cf",
+    GitSource("https://github.com/geodynamics/mineos.git", "3dd7c7433766d630b929d8254d03e705808ff8a3"),
+    DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd ${WORKSPACE}/srcdir/mineos
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-doc
-make
+
+# Drop docs build target
+atomic_patch -p1 ../patches/drop_docs.diff
+
+autoupdate
+autoreconf --install
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+
+# Fix clang error 'error: non-void function * should return a value [-Wreturn-type]'
+if [[ "${target}" != *-freebsd* ]] || [[ "${target}" == *-apple-* ]]; then
+    make CFLAGS="-Wno-return-type $cflags"
+else
+    make
+fi
```

Comment:
Much simpler and doesn't need to have multiple `make` calls
```suggestion
# Fix clang error 'error: non-void function * should return a value [-Wreturn-type]'
if [[ "${target}" != *-freebsd* ]] || [[ "${target}" == *-apple-* ]]; then
    export CFLAGS="-Wno-return-type"
fi
./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
make
```

Reply:
Thanks!!